src/test/rgw/rgw_multi/tests_ps.py

```diff
@@ -4400,340 +2179,3 @@ def test_ps_missing_topic():
 
     # cleanup
     master_zone.delete_bucket(bucket_name)
-
-
-def test_ps_s3_topic_update():
```

Comment:
keep this tests here, but we need an "on_master" version of it as well

Reply:
So, a new test needs to be added in new file similar to this?

Reply:
yes